app/views/reviews/show.html.erb

```diff
@@ -20,12 +20,25 @@
 
 <h3>Ratings</h3>
 
-<p>Was this review kind, specific, and actionable?</p><br />
+<p id='rating-criteria'>Was this review kind, specific, and actionable?</p><br />
+
+<div id='new-rating' class='load-form'></div>
 
 <div>
   <div class='mini-box' id='new-rating-box'>
     <div class='squarebrackets'>
-      <center><%= link_to "+ rate review", new_rating_path(@review), class: 'body-link new-form-link', remote: true, id: 'new-rating-link' %></center>
+      <center>
+        <div class='body-link'>
+          <p>+ rate review</p><br />
+          <%# <%= link_to "+ rate review", new_rating_path(@review), class: 'new-form-link', remote: true, id: 'new-rating-link' %>
```

Comment:
I think this would look better below the thumbs up and down, what do you think? It would be more in line with the rest of the cards, and having the + as the first thing might be confusing, with people thinking this is still a link. I think the plus is fine if this is below.